athenz-identity-provider-service/pom.xml

```diff
@@ -68,20 +111,19 @@
 
         <!-- TEST -->
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>zkfacade</artifactId>
+            <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.yahoo.vespa</groupId>
-            <artifactId>container-test</artifactId>
-            <version>${project.version}</version>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
```

Comment:
Please depend on hamcrest-core instead of hamcrest-all (embeds old junit)

Reply:
The code uses org.hamcrest.Matchers.containsInAnyOrder(), which is not available in that artifact. Might be possible to rewrite, though.

Reply:
Fixed